### DISCUSSION TITLE
Dual Solis S6 Inverters

### DISCUSSION THREAD

**Discussion by whiteas2** [Pho3niX90/solis_modbus#68] [Q&A]
Hi. 

Is it possible to use this with dual inverters? I am connected to master; but seems some figures are incorrect (for example the  power to load is around half of what it should be). If i look in developer tools > states; I don't see the expected value either.

I have tried to load a second instance for the 2nd inverter but get an 'unknown error'.

**Comment by japers** [comment]
I have two inverters and it works but intermittently. After some time (a few hours) one of the inverters will disconnect, I then have to try disable/enable or even deleting/reinstalling before it starts working again. 

I have the exact same problem with the Solax Modbus integration. 

My solution (still undergoing testing but stable for 36h) is to have a different integration connecting to separate inverters.
